### PR TITLE
Doc/kube use cases

### PR DESCRIPTION
**Pull request by szuecs**
add shadow traffic and adjust sections to be shown more nicely in our docs page

### CODE REVIEW

**Comment by szuecs** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

File: docs/dataclients/kubernetes.md [hunk 1]
```diff
@@ -203,7 +203,112 @@ something that configures your frontend loadbalancer, for example
 and your DNS, [external-dns](https://github.com/kubernetes-incubator/external-dns)
 automatically.
 
-## Blue-Green deployments
+## Skipper Ingress Annotations
+
+Annotation | example data | usage
+--- | --- | ---
+zalando.org/backend-weights | {"my-app-1": 80, "my-app-2": 20} | green-blue deployments
```
Comment (aryszka) [inline docs/dataclients/kubernetes.md]: shouldn't it be "blue-green" instead of "green-blue"?

File: docs/dataclients/kubernetes.md [hunk 2]
```diff
@@ -203,7 +203,112 @@ something that configures your frontend loadbalancer, for example
...
+### Basic Auth
+
+    % htpasswd -nbm myName myPassword
+
+    basicAuth("/path/to/htpasswd")
+    basicAuth("/path/to/htpasswd", "My Website")
+
+## Diagnosis - Throttling - Latency
+
+For diagnosis purpose there are filters that enables you to throttle
```
Comment (aryszka) [inline docs/dataclients/kubernetes.md]: "there are filters that enables" -> "there are filters that enable"

File: docs/dataclients/kubernetes.md [hunk 3]
```diff
@@ -240,6 +345,11 @@ and **my-app-2** will get **20%** of the traffic:
               servicePort: http
             path: /
 
+# Filters
+
+Filters can modify http requests and responses. There are a plenty of
```
Comment (aryszka) [inline docs/dataclients/kubernetes.md]: Potentially better but not sure: "There are plenty of things..." (without the "a")

**Comment by aryszka** [comment]
👍

**Comment by szuecs** [comment]
thanks for the review

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**